ocs_ci/utility/ibm_hci.py

```
@@ -0,0 +1,721 @@
+
+        Args:
+            rack_ip (str): IP address of the rack
+            username (str): SSH username
+            password (str): SSH password
+            command (str): Command to execute
+
+        Returns:
+            tuple: (stdout, stderr, exit_code)
+        """
+        ssh = SSHClient()
+        ssh.set_missing_host_key_policy(AutoAddPolicy())
```

Ruff (0.15.12)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)